man/systemd-analyze.xml

```diff
@@ -547,6 +554,31 @@ NAutoVTs=8
       </example>
     </refsect2>
 
+    <refsect2>
+      <title><command>systemd-analyze vercmp <replaceable>VERSION1</replaceable> <replaceable>VERSION2</replaceable></command></title>
+
+      <para>This command will compare the two specified version strings and print
+      either <literal><replaceable>VERSION1</replaceable> &lt; <replaceable>VERSION2</replaceable></literal>,
+      or <literal><replaceable>VERSION1</replaceable> == <replaceable>VERSION2</replaceable></literal>,
+      or <literal><replaceable>VERSION1</replaceable> &gt; <replaceable>VERSION2</replaceable></literal>
+      as appropriate.</para>
+
+      <para>The exit status is <constant>0</constant> if the versions are equal, <constant>11</constant> if
```

Comment:
So not sure we need it, but `dpkg --compare-versions` has a nice interface because it allows the user to specify what the expected result should be, so it's really nice and flexible, eg:

```
$ dpkg --compare-versions 3.2.1.0 ge 0.1.2.3; echo $?
0
$ dpkg --compare-versions 3.2.1.0 lt 0.1.2.3; echo $?
1
```

Do we maybe want the same here?